addons/sms/models/sms_api.py

```diff
@@ -12,9 +12,37 @@ class SmsApi(models.AbstractModel):
     _name = 'sms.api'
     _description = 'SMS API'
 
+    @api.model
+    def _send_multi_sms(self, messages):
+        """ Send SMS using IAP in batch mode
```

Comment:
I think this should be the role of this method to prepare the parameters. I would give sms.sms instead of messages list of dict as sms.sms model contains everything you need.

Reply:
It was for the possibility to only send sms, without creating sms.sms